core/src/main/kotlin/org/evomaster/core/parser/RegexHandler.kt

```diff
@@ -44,7 +45,7 @@ object RegexHandler {
      * This would throw an exception if regex is invalid, or if it
      * has features we do not support yet
      */
-    fun createGeneForPostgresLike(regex: String): RegexGene{
+    fun createGeneForPostgresLike(regex: String, databaseType: DatabaseType? = null): RegexGene {
 
```

Comment:
this is not what I meant :)
notice that the method `createGeneForPostgresLike` as `Postgres` inside its name.
so, if `databaseType` is Postgres, then we call this method.
Note that, for the moment, we only support Postgres.

Reply:
Sorry, I missed the "Postgres" in the method name. I will fix that.